pkg/config/config.go

```diff
@@ -207,6 +207,9 @@ type EngineConfig struct {
 	// memory.
 	EnablePortReservation bool `toml:"enable_port_reservation,omitempty"`
 
+	// Environment variables to be used when running the container engine. For example "http_proxy=internal.proxy.company.com"
+	Env []string `toml:"env,omitempty"`
```

Comment:
If it's not too late (and we haven't cut a release yet, I think) - why is this []string and not map[string]string?

Reply:
Environments are slices in the stdlib and commonly used like that across the stack. `pkg/env` allows to conveniently translate between slices and maps. Also, slices are much easier to configure in config file than maps.